SPEC.md

```diff
@@ -159,8 +159,7 @@ This example pod will use a set of three apps:
 
 #### Pod UUID
 
-The executor must assign an [RFC4122 UUID](http://www.ietf.org/rfc/rfc4122.txt) to each pod.
-This UUID is exposed to the pod through the [Metadata Service](#app-container-metadata-service).
+Each pod much be assigned an [RFC4122 UUID](http://www.ietf.org/rfc/rfc4122.txt). This UUID is exposed to the pod through the [Metadata Service](#app-container-metadata-service).
```

Comment:
Please retain the sentence-per-line format for now (we prefer it for reviewability).